### ISSUE TITLE
Linked items to drag at once within different drop zones

### ISSUE THREAD

**Issue by DavidM42** [isaacHagoel/svelte-dnd-action#353]
So I currently have multiple rows of dnd zones. In those zones are items oriented as columns.

What I'd now like to achieve is that some of these column items are linked to the column item above or below them in another dnd row.
When one of them is moved, the other one should start dragging to and be placed appropriately one below or above the other.

How could I go about doing that?

**Comment by isaacHagoel** [comment]
Have you looked at the multi drag example (link in the readme)?

**Comment by DavidM42** [comment]
Yes I did and I understand that it probably is done within the consider and finalize handlers as well as needing custom styling.

The linked example is a bit hard for me to understand if I'm honest, but I'll look into it more and hopefully understand how to transfer it to my use case.
I don't know if these kinds of multi drags could be a common use case and useful to natively support in the library

**Comment by isaacHagoel** [comment]
have you seen the multi-drag example?

**Comment by isaacHagoel** [comment]
Closing because the question has been answered or a workable solution is documented in the discussion, and no further library change is currently outstanding. If this still affects v0.9.77, please open a focused issue with a current reproduction.
